includes/dbFacile.php

```diff
@@ -17,6 +17,23 @@
  * 3. Oh, and dbFetchAll() is now dbFetchRows()
  */
 
+function dbConnect($host = null, $user = '', $password = '', $database = '', $port = null, $socket = null)
```

Comment:
Does it not make more sense to not pass all these params and just call $config[] from within the function?